README.adoc

```diff
@@ -553,11 +553,11 @@ When you are finished, exit dev mode by pressing `CTRL+C` in the command-line se
 
 == Running the application from a minimal runnable JAR
 
-So far, Open Liberty was running out of the `target/liberty/wlp` directory, which effectively contains an Open Liberty server installation and the deployed application. The final product of the Maven build is a server package for use in a continuous integration pipeline and, ultimately, a production deployment.
+So far, Open Liberty was running out of the `target/liberty/wlp` directory, which effectively contains an Open Liberty instance and the deployed application. The final product of the Maven build is a server package for use in a continuous integration pipeline and, ultimately, a production deployment.
 
-Open Liberty supports a number of different server packages. The sample application currently generates a `usr` package that contains the servers and application to be extracted onto an Open Liberty installation.
+Open Liberty supports a number of different server packages. The sample application currently generates a `usr` package that contains the runtime and application to be extracted onto an Open Liberty installation.
 
-Instead of creating a server package, you can generate a runnable JAR file that contains the application along with a server runtime. This JAR file can then be run anywhere and deploy your application and server at the same time. To generate a runnable JAR file, override the  `include` property: 
+Instead of creating a server package, you can generate a runnable JAR file that contains the application along with a runtime. This JAR file can then be run anywhere and deploy your application and runtime at the same time. To generate a runnable JAR file, override the  `include` property: 
```

Comment:
```suggestion
Instead of creating a server package, you can generate a runnable JAR file that contains the application along with a Liberty runtime. This JAR file can then be run anywhere and deploy your application and runtime at the same time. To generate a runnable JAR file, override the  `include` property: 
```